packages/cli/__tests__/auth/__system__/cli.auth.login.apiml.system.test.ts

```diff
@@ -144,7 +161,7 @@ describe("auth login/logout apiml create profile", () => {
        https://github.com/zowe/zowe-cli/issues/2005
     */
     // eslint-disable-next-line jest/no-disabled-tests
-    it.skip("TODO: After 2005 is fixed: should successfully issue the login command and create a team config", () => {
+    it.skip("TODO: After 2005 is fixed: should successfully login with password and create a team config", () => {
```

Comment:
It seems that 2005 was closed on Jan 31, 2024 🤯 

Is this something we should not skip anymore?

Reply:
This was a good suggestion. Simply re-enabled the test, and it worked.